packages/@aws-cdk/aws-apigatewayv2-integrations/README.md

```diff
@@ -26,6 +26,37 @@
 
 Integrations connect a route to backend resources. HTTP APIs support Lambda proxy, AWS service, and HTTP proxy integrations. HTTP proxy integrations are also known as private integrations.
 
+### AWS Service
```

Comment:
This classification of this being an AWS service is probably not needed. 

Let's just call this section `Event Bridge`. Whether it's an AWS service integration or a private integration is an implementation detail that the CDK can hide.

Reply:
👍  e76381435